Library/Homebrew/cmd/update-report.rb

```diff
@@ -253,6 +271,85 @@ def migrate_cache_entries_to_double_dashes(initial_version)
     end
   end
 
+  def migrate_cache_entries_to_symlinks(initial_version)
```

Comment:
I'm wondering about getting some of the `update-report` logic into another file; it's getting pretty huge.

Reply:
Yep.

Reply:
@reitermarkus Cool, that'd be good, thanks 👍

Reply:
@reitermarkus Maybe just all the various migrations.